submissions/sinchUK/HTML-CSS-Popup/index.html

```diff
@@ -0,0 +1,204 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Roboto&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="./styles.css">
+    <title>Popup</title>
+</head>
+<body>
+    <header class="menu">
+        <nav>
+            <ul class="menu-items">
+                <li class="menu-item">
+                    <a href="#">
+                        Mail
+                    </a>
+                </li>
+                <li class="menu-item">
+                    <a href="#">
+                        Images
+                    </a>
+                </li>
+                <li class="menu-item">
+                    <div class="menu-item__inner">
+                        <input class="menu-item__input" type="checkbox"/>
+                        <img class="menu-item-img" src="./icons/popup-button.png" alt="submenu-icon"/>
+                        <div class="tooltip">
+                            <div class="tooltip-items">
```

Comment:
If you have a nested navigation use a nested list of links:
```HTML
    <nav>
        <ul>
            <li><a href="#">Link-1</a></li>
            <li><a href="#">Link-2</a></li>
            <li><a href="#">Link-3</a></li>
            <li><ul>
                <li><a href="#"></a>Link-1</li>
                <li><a href="#"></a>Link-2</li>
                <li><a href="#"></a>Link-3</li>
                <li><a href="#"></a>Link-4</li>
            </ul></li>
            <li><a href="#">Link-5</a></li>
        </ul>
    </nav>
```

Reply:
I changed code and used unordered list tag.
![Screenshot 2022-10-20 133703](https://user-images.githubusercontent.com/17921586/196926375-dff5d279-88ec-4ad6-a314-3d5c29e59de6.png)